mitosheet/src/components/taskpanes/DropDuplicates/DropDuplicates.tsx

```diff
@@ -235,21 +230,21 @@ const DropDuplicatesTaskpane = (props: DropDuplicatesProps): JSX.Element => {
                         ) 
                     })}
                 </MultiToggleBox>
-                {loading &&
-                    <Row className='mt-5'>
-                        <p className='text-subtext-1'>
-                            Deduplicating {<LoadingDots/>}
-                        </p>
-                    </Row>
-                }
-                {!loading &&
-                    <Row className='mt-5'>
-                        <p className='text-subtext-1'>
-                            Removed {originalNumRows - props.sheetDataArray[params.sheet_index].numRows} rows
-                        </p>
-                    </Row>
-                }
-                
+                <Spacer px={25}/>
+                <TextButton
+                    variant='dark'
+                    width='block'
+                    onClick={edit}
+                    disabled={false}
+                >
+                    {!editApplied 
+                        ? `Drop duplicates across ${params.column_ids.length} columns` 
+                        : (loading 
+                            ? 'Dropping...' 
+                            : `Removed ${result?.num_rows_dropped || 0} rows`
+                        )
+                    }
+                </TextButton>
```

Comment:
> for example, have both "June" and "Jun" in their dataset, correcting them and then trying to deduplicate again.

This specific example is actually impossible, because of the fact that we close the taskpane if the user goes to fix up their data. So perhaps waiting to make changes until we fix up this technical debt makes more sense.

Both deduplicate and fill nan are imdepotent operations (clicking the button again without changing the params doesn't do anything) so there isn't a difference in these cases between not sending a message and just overwriting with the same change. 

This isn't the case with split text to column, so I personally vote we wait to see how the hook works with that, and if it's intuitive to make changes to this behavior. If you want to change it in that PR, go for it.

Re: moving the result communication below the button. This seems like an independent question. Would you like me to do that? Happy to (although I am happy with it as is, your call).

Reply:
> Both deduplicate and fill nan are imdepotent operations

They are, but when there is a bug, like there is now in the closing of editing taskpanes, the assumption that if you don't change the configuration then nothing will change breaks. The current approach seems a bit fragile to me.

My vote would be that we move the results below the button, but that if the user clicks the button again, it changes to 0 rows were removed, as to give the user feedback about what is happening.